configs/RCCS-Fugaku-Fujitsu-A64FX-TofuD/spack.yaml

```diff
@@ -6,20 +6,31 @@
 spack:
   packages:
     default-compiler:
-      spack_spec: fj@{default_fj_version}
-    default-mpi:
-      spack_spec: fujitsu-mpi@{default_fj_version}%fj arch=linux-rhel8-a64fx
-    compiler-gcc:
+      #spack_spec: fj@{default_fj_version}
+      #spack_spec: clang@{default_llvm_version}
```

Comment:
Can you remove these?

Reply:
in theory yes, but i would like to keep them for now, because i regularly switch compilers to test things and right onw the "default" is gcc because i was able to build amg and gromacs with it. are those comments causing issues or why would you want them to go?

Reply:
I thought this was leftover from debugging, but given that you explain this is useful in the future, it seems fine to keep for now. We should work on providing another mechanism for you to make this switch without having to manually edit.